File: Java/Week 3/Palindrome.java
class Palindrome {
    public static void main(String[] args) {
       int n = 523425, rev= 0;
       int x = n;
       while(n>0){
           rev=rev*10+(n%10);
           n=n/10;
       }
       if(rev==x){
           System.out.println("Palindrome");
       } else{
           System.out.println("Not Palindrome");
       }
    }
}

File: Python/Week 3/palindrome.py
n = 3533
// We are copying n because n is going to be manipulated to find each digit in it so at the end it will become 0.
x = n
revNum = 0
while n > 0:
    lastDigit = n % 10
    revNum = (revNum * 10) + lastDigit
    n = n // 10
if revNum == x:
    print("Palindrome")
else:
    print("Not Palindrome")
